runtime/src/main/java/org/corfudb/runtime/view/replication/ReadWaitHoleFillPolicy.java

```diff
@@ -74,8 +74,10 @@ public ILogData peekUntilHoleFillRequired(long address, Function<Long, ILogData>
                 }
 
                 // Otherwise try again.
-                log.debug("peekUntilHoleFillRequired: Attempted read at address {}, "
-                        + "but data absent. Retrying.", address);
+                if (log.isTraceEnabled()) {
```

Comment:
please don't use `if` condition http://www.slf4j.org/faq.html#logging_performance

Reply:
done, good to know! thanks